pkg/service/capabilities/service.go

```diff
@@ -0,0 +1,240 @@
+package capabilities
```

Comment:
Moved these here so the replicator can depend on these methods without a circular dep (it can't depend on the storage service)

Reply:
I can't decide if these are really a whole "service" -- they really have no internal mutable state.

I almost wonder if you want to instead do this:

- move to something like `pkg/service/storage/handlers/blobs/blobs.go` 
- define the service not as a struct but as an interface with the deps you need (ID()/Blobs()/PDP()/Claims())
- hey the storage service already matches the interface!

- change BlobAllocate / BlobReplicate to pure functions taking the service as the first dependency

- avoids ever having to have a seperate capabilities service (btw, if you do want to keep the service, I would rename it "handlers" or something)